keda/src/components/scaledjobs/Detail.tsx

```diff
@@ -0,0 +1,198 @@
+import {
+  ConditionsSection,
+  DetailsGrid,
+  NameValueTable,
+  SectionBox,
+} from '@kinvolk/headlamp-plugin/lib/components/common';
+import Job from '@kinvolk/headlamp-plugin/lib/k8s/job';
+import { useParams } from 'react-router-dom';
+import { useKedaInstalled } from '../../hooks/useKedaInstalled';
+import { ScaledJob } from '../../resources/scaledjob';
+import {
+  ContainersSection,
+  JobsListRenderer,
+  NotInstalledBanner,
+  TriggersSection,
+} from '../common/CommonComponents';
+
+export interface OwnedJobsSectionProps {
+  scaledJob?: ScaledJob;
+  hideColumns?: string[];
+  /**
+   * Hides the namespace selector
+   */
+  noSearch?: boolean;
+}
+
+export function OwnedJobsSection(props: OwnedJobsSectionProps) {
+  const { scaledJob, hideColumns, noSearch } = props;
+
+  const { items: jobs, errors } = Job.useList({
+    namespace: scaledJob?.metadata?.namespace,
+    labelSelector: `scaledjob.keda.sh/name=${scaledJob?.metadata?.name}`,
+  });
+
+  const onlyOneNamespace = !!scaledJob?.metadata?.namespace;
+  const hideNamespaceFilter = onlyOneNamespace || noSearch;
+
+  return (
+    <JobsListRenderer
+      hideColumns={
+        hideColumns.length > 0 ? hideColumns : onlyOneNamespace ? ['namespace'] : undefined
+      }
+      jobs={jobs}
+      errors={errors}
+      noNamespaceFilter={hideNamespaceFilter}
+    />
+  );
+}
+
+export function ScaledJobDetail() {
```

Comment:
For consistency lets follow the Overview, Spec, Status order in the details view. In this the conditions section is displayed first and then the spec etc.